### ISSUE TITLE
could not analyze the data: could not analyzeVersion: could not match version! check the versionCfg.json file

### ISSUE THREAD

**Issue by leopku** [yeqown/go-qrcode#12]
```go
url := "https://openapi.alipaydev.com/gateway.do?app_id=2016091200494382&biz_content=%7B%22out_trade_no%22%3A%22GZ201901301040361014%22%2C%22product_code%22%3A%22QUICK_WAP_WAY%22%2C%22quit_url%22%3A%22https%3A%2F%2Fwww.gopay.ink%22%2C%22subject%22%3A%22%E6%89%8B%E6%9C%BA%E7%BD%91%E7%AB%99%E6%B5%8B%E8%AF%95%E6%94%AF%E6%8C%81%22%2C%22total_amount%22%3A%220.01%22%7D&charset=utf-8&format=JSON&method=alipay.trade.wap.pay&notify_url=https%3A%2F%2Fwww.gopay.ink&sign=kD1ezhNwjkA6PZzYOvvNd5wTtOeAo5kgCxXaeZ6nhNyB4g3nDndy5hAwQeWVBVlTJiL9MD%2Bn%2BhOTghZVqizNrmA2MB4eTAIMFjmM6yk0jAG6Rug3updNtDLBD30sDP%2FEwhi2lsliEg6Z3L0wfYgXdjpR%2B%2Bl1xCP2t3Bs8LYbiDPLBSibatDpLv7HZErH0jA5nMqolYj%2F92Fsdb%2BxS%2Fdcz7aqUHcEeVzdG%2FV8bQxRt0VzPe5hOO173BpPtolsrYZCVJrVoH3aYa5fW%2BDxTo3BVZkroCtjsh4mkdPaXac6RGhAE%2BR7H%2BF514OlzaxiDHNKMQaktfO6haipRYta3ejo2g%3D%3D&sign_type=RSA2&timestamp=2021-01-07+17%3A24%3A52&version=1.0"

		qrc, err := qr.New(payUrl)
		if err != nil {
			log.Error().Err(err).Msg("")
			return
		}
```
Would failed with 
```
5:26PM ERR cmd/alipay.go:73 >  error="could not analyze the data: could not analyzeVersion: could not match version! check the versionCfg.json file"
```

**Comment by yeqown** [comment]
There are not full version configs of the QR protocol, I would take some days to complete this (this has been forgotten too long..) 😂

**Comment by leopku** [comment]
@yeqown Thanks for your reply.

I found that if changing url to a shorter string, this bug would not occurred.

**Comment by yeqown** [comment]
https://www.qrcode.com/zh/about/version.html

**Comment by leopku** [comment]
> https://www.qrcode.com/zh/about/version.html

I'm not export of QRCode. 
Thanks for your link so I can learn it deeply.

**Comment by yeqown** [comment]
@leopku This is my mistake, I did not finish all version of QR code, so it could not bear a string which is longer than version 5 could bear.

**Comment by yeqown** [comment]
@leopku I would fill all versions tomorrow ~, plz wait for one day.

**Comment by yeqown** [comment]
#13 has fixed this issue, you can try on `v1.4.1` to verify, also reopen if you got any problems.

**Comment by leopku** [comment]
@yeqown It works now.
Thanks.
